js/pages/tola_management_pages/country/components/edit_disaggregations.js

```diff
@@ -411,7 +411,7 @@ export class DisaggregationType extends React.Component {
         const managed_data = this.state;
         const retroPrograms = managed_data.id === "new" && Object.values(this.programsForRetro).length > 0 ? <RetroProgramCheckBoxWrapper
                 programs={this.programsForRetro}
-                disabled={this.state.selected_by_default !== true || this.props.formDisabled}
+                disabled={this.state.selected_by_default !== true}
```

Comment:
just curious, trying to follow the logic, what is replacing the previous "props.formDisabled"?  Is it because the modal form of the notice blocks the form so we don't need to duplicate by disabling?

Reply:
Yep, exactly.